snapcraft/how-to/interface-management.md

```diff
@@ -34,61 +29,36 @@ slots:
-
-![Snap store permissions](https://assets.ubuntu.com/v1/7fbcf74c-store-04.png)
-
-Each interface can now be connected or disconnected by selecting the toggle switch to the right of its description, and you may be prompted for your password.
-
--->
-
-<h2 id='heading--listing'>Snap connections</h2>
+## View all the interfaces used by a snap
 
-On the terminal, the _snap_ command provides more granular control over interface connections and which interfaces are operational on your system.
-
-The `snap connections` command lists which interfaces are connected and being used, while adding `--all` additionally shows interfaces with unconnected slots or plugs (shown in the output as a `-`):
+Run the command:
 
 ```bash
-$ snap connections --all
-Interface            Plug                           Slot                     Notes
-adb-support          scrcpy:adb-support             :adb-support             -
-alsa                 ffmpeg:alsa                    :alsa                    manual
-appstream-metadata   snap-store:appstream-metadata  :appstream-metadata      -
-iaudio-playback      ardour:audio-playback          :audio-playback          -
-dbus                 -                              cameractrls:dbus-daemon  -
-[...]
+snap connections <snap name>
 ```
 
-To see which interfaces a snap is using, and which interfaces it could use but isn't, type `snap connections <snapname>`:
+The `connections` command lists the interfaces a snap uses. For example, for the VLC snap you'd see:
 
 ```bash
 $ snap connections vlc
-Interface       Plug                   Slot                 Notes
-audio-playback  vlc:audio-playback     :audio-playback      -
-audio-record    vlc:audio-record       -                    -
-camera          vlc:camera             -                    -
-desktop         vlc:desktop            :desktop             -
-home            vlc:home               :home                -
-(...)
+Interface         Plug                  Slot               Notes
+camera            vlc:camera            -                  -
+desktop           vlc:desktop           :desktop           -
+desktop-legacy    vlc:desktop-legacy    :desktop-legacy    -
+home              vlc:home              :home              -
+mount-observe     vlc:mount-observe     -                  -
+[...]
 ```
 
-In the above output, the [`camera`](/t/the-home-interface/7838) interface is not connected because its slot is empty. This means VLC cannot access any connected cameras.
+In the previous <!-- above, left, right, below are assuming reading direction --> example, you <!-- TODO Style guide question, as authors should we talk directly to our readers YOU instead of WE --> can see that the `vlc:camera` interface is disconnected because it has an empty *Slot* entry.
```

Comment:
Good question. There's no firm guide to this - both _we_ and _you_ are acceptable, but I prefer avoiding them when possible, eg. "In the previous example, the output shows that the `vlc:camera`..."